Gemfile

```diff
@@ -12,6 +12,7 @@ gem "activerecord-session_store",      "~>0.1.2", :require => false
 # Local gems
 path "gems/" do
   gem "manageiq_foreman", :require => false
+  gem "provider_amazon"
```

Comment:
I'm not so sure about the naming...
_provider_ prefix is ok, but it might be better to use _miq_provider ?
Also, maybe 'aws' instead of amazon?